README.md

```diff
@@ -1,7 +1,7 @@
 # Tendermint Spec
 
-This repository contains the paper describing the Tendermint consensus
-algorithm, including formal proofs of its safety and liveness.
+This repository contains specs of the current and future implementation of Tendermint the protocol, papers that have been published by the ICF or AIB and ongoing research in a variety of areas.
 
-For the pdf, see the [latest
-release](https://github.com/tendermint/spec/releases).
+The paper describing the Tendermint consensus algorithm, including formal proofs of its safety and liveness.
```

Comment:
Maybe this sentence can be removed or moved to paper folder.

Reply:
will remove this and once move the pdf version into papers/ in a following PR